### PR TITLE
Fix for Retire Common Crawl DAGs

### LINKED ISSUE #846: Retire Common Crawl DAGs
## Description

We have two Common Crawl related DAGs available in production but not enabled: `commoncrawl_etl_workflow` and `sync_commoncrawl_workflow`. We currently don't have re-enabling them as part of our future priorities, so it might be best just to retire them for now to de-clutter the DAGs view. We'll still keep the code, but we would no longer need to navigate around them in the UI :slightly_smiling_face:

I believe we could move the entirety of the `dags/commoncrawl/` module into the retired folder to address this.

@WordPress/openverse-catalog - any concerns about doing this?

## Implementation

- [ ] 🙋 I would be interested in implementing this feature.

### PR DESCRIPTION
**Pull request by Pmeet**
## Fixes

Fixes #846 by @AetherUnbound 

## Description

This pull request has made the following changes
1. Retired the dags/commoncrawl module to retired folder
2. Made changes to some tests and airflowignore file in accordance with the above change

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [x] My pull request targets the *default* branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [x] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: tests/dags/common/etl/test_commoncrawl_utils.py [hunk 1]
```diff
@@ -1,8 +1,12 @@
+# Retiring the dags/commoncrawl module so retiring this test file
```
Comment (AetherUnbound) [inline tests/dags/common/etl/test_commoncrawl_utils.py:1]: This file can be removed entirely!

File: tests/dags/test_dag_parsing.py [hunk 2]
```diff
@@ -18,12 +18,12 @@
...
     "maintenance/airflow_log_cleanup_workflow.py",
     "maintenance/pr_review_reminders/pr_review_reminders_dag.py",
-    "commoncrawl/sync_commoncrawl_workflow.py",
-    "commoncrawl/commoncrawl_etl.py",
     "database/recreate_popularity_calculation_dag_factory.py",
     "data_refresh/dag_factory.py",
     "oauth2/authorize_dag.py",
     "oauth2/token_refresh_dag.py",
+    #    "commoncrawl/sync_commoncrawl_workflow.py", # retired
+    #    "commoncrawl/commoncrawl_etl.py", #retired
```
Comment (AetherUnbound) [inline tests/dags/test_dag_parsing.py:26]: These lines can also just be removed
```suggestion
```

**Review by AetherUnbound — COMMENTED** [review]
Thanks for this PR @Pmeet! I have a few requests for changes, we generally don't keep test fragments when we retire things 😄 Also, it looks like CI is failing:
> `!! Changes found in DAG documentation, please run 'just generate-dag-docs' locally and commit difference !!`

File: tests/dags/test_dag_parsing.py (hunk 2 above)
Comment (Pmeet) [reply tests/dags/test_dag_parsing.py]: Hi @AetherUnbound Thanks for informing of these changes I'll open a new PR for the same.

Comment (Pmeet) [reply tests/dags/test_dag_parsing.py]: Hi @AetherUnbound , I just committed the changes you mentioned. Let me know if any other changes are required and I am fairly new at Open source  contributions in general so please feel free to let me know if any mistakes I made.

**Comment by AetherUnbound** [comment]
Ah drat, it looks like #866 caused some merge conflicts 😓 I think the best way to resolve the conflicts might be to:

1. Open a new PR with the same changes done, but this time off of an updated version of `origin/main` or
2. I can open a PR using your commits and give you co-author credit while resolving the git conflict

I'm fine either way, the first option ensures you're still the PR submitter while option 2 is less work for you while still giving you credit. Which would you prefer @Pmeet? 🙂

**Comment by Pmeet** [comment]
> Ah drat, it looks like #866 caused some merge conflicts 😓 I think the best way to resolve the conflicts might be to:
> 
> 1. Open a new PR with the same changes done, but this time off of an updated version of `origin/main` or
> 2. I can open a PR using your commits and give you co-author credit while resolving the git conflict
> 
> I'm fine either way, the first option ensures you're still the PR submitter while option 2 is less work for you while still giving you credit. Which would you prefer @Pmeet? 🙂

I think you should do it since you are more experienced in handling this type of conflict. I'm afraid that since I'm new I'll break something somehow hehe.

**Comment by AetherUnbound** [comment]
Okay! I've made #870 and it looks like you were attributed correctly in the commits 😄 I'm going to go ahead and close this one in favor of the other, thanks for the contribution!

**Pull request closed without merging**